en_US/advanced/cli.md

```diff
@@ -1409,11 +1409,30 @@ ok
   ok
   ```
 
-+ acl del _all `<Topic>`
++ acl del `_all` `<Topic>`
 
   Delete $all acl
 
   ```bash
   ./bin/emqx_ctl acl del _all Topic/C
   ok
-  ```
\ No newline at end of file
+  ```
+
+## pem_cache command
+
+The `pem_cache` command is used to clear the cache of all SSL-related certificates.
+For example, after replacing the SSL listener's certificate file, you can use this command
+to make the new certificate file take effect instead of restarting EMQX's SSL listener.
```

Comment:
```suggestion
to make the new certificate file take effect instead of restarting EMQX's SSL listener.

**NOTE** The updates of the cache do not affect existing TLS connections.
```